src/www/ui/api/documentation/openapi.yaml

```diff
@@ -1263,6 +1495,72 @@ paths:
 
         default:
           $ref: '#/components/responses/defaultResponse'
+
+  /uploads/{id}/item/{itemId}/copyrights:
+    parameters:
+      - name: id
+        required: true
+        description: Upload Id
+        in: path
+        schema:
+          type: integer
+      - name: ItemId
+        required: true
+        description: UploadTree ID
+        in: path
+        schema:
+          type: integer
+      - name: status
+        required: true
+        in: query
+        description: status of the copyright
+        schema:
+          type: string
+          enum:
+              - active
+              - inactive
+      - name: limit
+        description: Limits of responses per request
+        required: false
+        in: header
+        schema:
+          type: integer
+          default: 100
+          minimum: 1
+          maximum: 1000
+      - name: page
+        description: Page number for responses
+        required: false
+        in: header
+        schema:
+          type: integer
+          default: 1
+          minimum: 1
+          maximum: 1000
```

Comment:
Why the max value of 1000?

Reply:
Should I be removing the limit parameter from the docs?

Reply:
Its an arbitrary value and thus should not be there.